scss/navbar.scss

```diff
@@ -9,6 +9,7 @@
   top: 0;
   background: white;
   border-top: 6px solid $cerulean;
+  z-index: 99999999;
```

Comment:
what if you then need a popup in front of the navbar? 😬 
It is good practice to keep track of all of your z-indexes in a file so that if you need to add something with a custom z-index you know what z-index to use